lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.h

```diff
@@ -335,6 +335,10 @@ class TypeSystemSwiftTypeRef : public TypeSystemSwift {
   clang::api_notes::APINotesManager *
   GetAPINotesManager(ClangExternalASTSourceCallbacks *source, unsigned id);
 
+  /// Check whether the type being dealt with is tricky to validate due to
```

Comment:
Could we wrap the entire function in #ifndef NDEBUG so we don't accidentally call it in non-asserts builds?